src/main/java/duke/TaskManager.java

```diff
@@ -0,0 +1,93 @@
+package duke;
+
+import duke.command.*;
```

Comment:
Perhaps, you can consider listing out imported classes explicitly instead of using wild card imports. This helps you and other developers easily understand the code base when all the classes have been grouped in packages.